test/test_ripser.py

```diff
@@ -45,6 +45,18 @@ def test_non_square_dist_matrix(self):
 
         with pytest.raises(Exception):
             ripser(data, distance_matrix=True)
+    
+    def test_too_many_perm(self):
```

Comment:
Could you also test with `n_perm=0` and `n_perm=None` and with default `n_perm`.